tests/test_filterLabelFixups.py

```diff
@@ -66,14 +66,9 @@ def setUp(self):
         # Parameters with bbox to test that logic still works on subimage gets.
         self.parameters = {"bbox": Box2I(Point2I(0, 0), Point2I(8, 7))}
 
-    def _skipMock(self, datasetType="_mock_calexp"):
+    def _skipMock(self, datasetType="calexp"):
```

Comment:
This test case uses a helper method but the other tests use skip_mock directly.

Maybe consider putting this method in a test case helper class in your tests/mock.py and adding that class to the testcase as a second inheritance class. Then all the test cases can use the method form directly and it's all consistent (I guess so long as `self.butler` exists -- maybe it doesn't, I haven't checked every place).

Reply:
This method already existed, so I decided to keep it around instead of replacing every use of it.

I was thinking about implementing `skip_test` as a method in a mixin class, but I was not comfortable with requiring `self.butler` in every test class. Maybe I should do it anyways as all current users do define `self.butler` already.

Reply:
I guess it could take an optional `butler: Butler = None` parameter that could override using `self.butler` if you were worried about that sort of thing.

Reply:
I have re-implemented it with a mixin class.

Reply:
Looks good.